protocol/0031-ETHB-ethereum_bridge_spec.md

```diff
@@ -148,3 +148,98 @@ This example connects the network to Ropsten:
 * ERC20 smart contract specific requirements:
   * A valid multisig bundle can be passed to the `remove_asset` function to successfully remove a previously  allow listed token (<a name="0031-COSMICELEVATOR-019" href="#0031-COSMICELEVATOR-019">0031-COSMICELEVATOR-019</a>)
   * An invalid multisig bundle is rejected by the `remove_asset` function (<a name="0031-COSMICELEVATOR-020" href="#0031-COSMICELEVATOR-020">0031-COSMICELEVATOR-020</a>)
+
+
+## ERC20 Bridge Logic to Vega Integration Tests (Vega System Tests)
+1.  `address payable public erc20_asset_pool_address;`
+   * must match the deployed asset pool address (<a name="0031-ETHB-018" href="#0031-ETHB-018">0031-ETHB-018</a>)
+2.  `function list_asset(address asset_source,bytes32 vega_asset_id,uint256 lifetime_limit,uint256 withdraw_threshold,uint256 nonce,bytes memory signatures)`
+   * must list asset (<a name="0031-ETHB-019" href="#0031-ETHB-019">0031-ETHB-019</a>)
+   * must not list non-asset (<a name="0031-ETHB-020" href="#0031-ETHB-020">0031-ETHB-020</a>)
+   * must not list if sigs bad (<a name="0031-ETHB-021" href="#0031-ETHB-021">0031-ETHB-021</a>)
+   * must not list if already listed (<a name="0031-ETHB-022" href="#0031-ETHB-022">0031-ETHB-022</a>)
+3.  `function remove_asset(address asset_source,uint256 nonce,bytes memory signatures)`
```

Comment:
What happens with assets "locked" on the bridge if this is called?

Reply:
removing asset only stops deposits